src/ai/backend/manager/models/gql.py

```diff
@@ -1875,16 +1875,23 @@ async def resolve_vfolder(
         user_id: Optional[uuid.UUID] = None,
     ) -> Optional[VirtualFolder]:
         graph_ctx: GraphQueryContext = info.context
-        user_role = graph_ctx.user["role"]
         loader = graph_ctx.dataloader_manager.get_loader(
             graph_ctx,
             "VirtualFolder.by_id",
```

Comment:
I want to use `get_loader_by_func` and `VirtualFolder.batch_load_by_id` here, but I'm getting a type error because the return type of `VirtualFolder.batch_load_by_id` differs from what `get_loader_by_func` expects.